docs/metaed-project/DomainEntity/User.metaed

```diff
@@ -1,38 +1,38 @@
-﻿// SPDX-License-Identifier: Apache-2.0
-// Licensed to the Ed-Fi Alliance under one or more agreements.
-// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
-// See the LICENSE and NOTICES files in the project root for more information.
-
 Domain Entity User
```

Comment:
I hesitate to ask this... but after writing SQL scripts with `User`, I've realized the annoyance of having to write `[User]` everywhere since "user" is a reserved word. Using the square bracket really isn't a big deal - except that it is not portable to PostgreSQL, so any queries with the square brackets must be rewritten to remove them for PostgreSQL. So... how difficult would it be to change this to `LMSUser` instead of `User`?

Reply:
I don't think this is a big lift. I will need to update the User MetaEd file and any references, the UDM, and the documentation in techdocs. I would also need to recreate the manual changes to the supporting-artifacts for the Identity fields.

@ejansson, do you see any issues with making this change?

Reply:
We could do that in a separate ticket.

Reply:
Updated User to LMSUser.